src/Microsoft.ML.AutoML/ColumnInference/PurposeInference.cs

```diff
@@ -115,11 +115,10 @@ private static class Experts
             internal sealed class TextClassification : IPurposeInferenceExpert
             {
                 private static readonly string[] _commonImageExtensions = { ".bmp", ".dib", ".rle", ".jpg", ".jpeg", ".jpe", ".jfif", ".gif", ".tif", ".tiff", ".png" };
+                public static string ImageFolder { private get; set; }
```

Comment:
@justinormont made a great point the other day --
Perhaps the user should pass this in through ExperimentSettings https://github.com/dotnet/machinelearning/blob/e50c4d20012e0d62852f404ae443afca7dad043e/src/Microsoft.ML.AutoML/API/ExperimentSettings.cs#L14 , and then experiment settings is available to be piped throughout the rest of the code